modules/nextflow/src/main/resources/META-INF/plugins-info.txt

```diff
@@ -1,8 +1,8 @@
 nf-amazon@1.11.0
-nf-wave@0.5.2
-nf-console@1.0.4
-nf-google@1.4.4
 nf-azure@0.14.2
-nf-tower@1.5.5
+nf-codecommit@0.1.2
 nf-ga4gh@1.0.4
-nf-codecommit@0.1.2
\ No newline at end of file
+nf-wave@0.5.2
+nf-console@1.0.4
+nf-tower@1.5.6
+nf-google@1.4.5
```

Comment:
This is another generated file. We only commit in the final release otherwise it mess up merges. Better to rollback to the previous version that should be the following 

```
nf-amazon@1.11.0
nf-wave@0.5.2
nf-console@1.0.4
nf-google@1.4.4
nf-azure@0.14.2
nf-tower@1.5.5
nf-ga4gh@1.0.4
nf-codecommit@0.1.2
```